docs/DeveloperPortal.md

```diff
@@ -33,6 +33,41 @@ end
 app = Spaceship.app.create!(bundle_id: "com.krausefx.app_name", name: "fastlane App")
 ```
 
+### App Services
+
+App Services are part of the application, however, they are one of the few things that can be changed about the app once it has been created.
+
+```ruby
+# Find a specific app based on the bundle identifier
+app = Spaceship.app.find("com.krausefx.app")
+
+# Enable HealthKit, but make sure HomeKit is disabled
+app.update_service(Spaceship.app_service.health_kit.on)
+app.update_service(Spaceship.app_service.home_kit.off)
```

Comment:
We should add a list of available services somewhere in the documentation. What do you think?